pkg/subctl/cmd/join.go

```diff
@@ -451,13 +454,15 @@ func populateSubmarinerSpec(subctlData *datafile.SubctlData, netconfig globalnet
 		Namespace:                SubmarinerNamespace,
 		CableDriver:              cableDriver,
 		ServiceDiscoveryEnabled:  subctlData.ServiceDiscovery,
+		ImageOverrides:           getImageOverrides(),
 	}
 	if netconfig.GlobalnetCIDR != "" {
 		submarinerSpec.GlobalCIDR = netconfig.GlobalnetCIDR
 	}
 	if len(customDomains) > 0 {
 		submarinerSpec.CustomDomains = customDomains
 	}
+
```

Comment:
I didn't notice this unrelated whitespace, not very important, but if you need to resubmit, please remove it.
```suggestion
```

Reply:
Done